src/fragment-components/a-progress-indicator.tsx

```diff
@@ -0,0 +1,205 @@
+				labelText="Label"
+				onChange={(event: any) => handleStepUpdate('label', event.currentTarget.value, index)} />
+			<TextInput
+				light
+				value={item.secondaryLabel}
+				labelText='Secondary label'
+				onChange={(event: any) => handleStepUpdate('secondaryLabel', event.currentTarget.value, index)} />
+			<div style={{ display: 'flex' }}>
+				<Checkbox
+					style={{ display: 'inline-flex' }}
+					labelText='Is invalid'
+					id={`invalid-select-${index}`}
+					checked={item.invalid}
+					onChange={(checked: any) => handleStepUpdate('invalid', checked, index)} />
+				<Checkbox
+					style={{ display: 'inline-flex' }}
+					labelText='Is disabled'
+					id={`disabled-select-${index}`}
+					checked={item.disabled}
+					onChange={(checked: any) => handleStepUpdate('disabled', checked, index)} />
+			</div>
+		</>;
+	};
+
+	const updateStepList = (newList: any[]) => {
+		setComponent({
+			...selectedComponent,
+			progressSteps: newList
+		});
+	};
+
+	return <>
+		<Checkbox
+			labelText='Is vertical'
+			id='layout-select'
+			checked={selectedComponent.isVertical}
+			onChange={(checked: any) => {
+				setComponent({
+					...selectedComponent,
+					isVertical: checked
+				});
+			}}
+		/>
+		<DraggableTileList
+			dataList={[...selectedComponent.progressSteps]}
+			setDataList={updateStepList}
+			updateItem={handleStepUpdate}
+			defaultObject={{
+				label: 'Step',
+				description: 'Description',
+				secondaryLabel: 'Optional label',
+				invalid: false,
+				disabled: false
+			}}
+			template={template} />
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent} />
+	</>;
+};
+
+export const AProgressIndicatorCodeUI = ({ selectedComponent, setComponent }: any) => {
+	return <TextInput
+		value={selectedComponent.codeContext?.name}
+		labelText='Input name'
+		onChange={(event: any) => {
+			setComponent({
+				...selectedComponent,
+				codeContext: {
+					...selectedComponent.codeContext,
+					name: event.currentTarget.value
+				}
+			});
+		}}
+	/>;
+};
+
+export const AProgressIndicator = ({
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		headingCss={css`display: block;`}
+		className={css`position: relative; display: flex`}
+		{...rest}>
+			<ProgressIndicator
+			currentIndex={componentObj.currentIndex}
+			vertical={componentObj.isVertical}
+			className={componentObj.cssClasses?.map((cc: any) => cc.id).join(' ')}>
+				{
+					componentObj.progressSteps.map((step: any, index: number) => (
+						<ProgressStep {...step} key={index} />
+					))
+				}
+			</ProgressIndicator>
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: AProgressIndicator,
+	styleUI: AProgressIndicatorStyleUI,
+	codeUI: AProgressIndicatorCodeUI,
+	type: 'progress-indicator',
+	keywords: ['progress', 'indicator'],
+	name: 'Progress indicator',
+	defaultComponentObj: {
+		type: 'progressIndicator',
```

Comment:
```suggestion
		type: 'progress-indicator',
```
?

After pulling changes and all...